es74x/build.gradle

```diff
@@ -52,11 +52,11 @@ compileScala {
 
 dependencies {
     compile project(path: ':core')
-
-    compile 'org.elasticsearch:elasticsearch:' + esVersion
-    compile 'org.elasticsearch.plugin:transport-netty4-client:' + esVersion
-
-    compile group: 'org.apache.logging.log4j',              name: 'log4j-core',           version: '2.11.0'
+    compile             group: 'org.elasticsearch',                 name: 'elasticsearch' ,             version: esVersion
+    compileClasspath    group: 'org.codelibs.elasticsearch.lib',    name: 'plugin-classloader',         version: esVersion
+    compileClasspath    group: 'org.locationtech.spatial4j',        name: 'spatial4j',                  version: '0.7'
```

Comment:
do we need it?

Reply:
I got runtime error without this dependency.